src/components/basket-item/basket-item.jsx

```diff
@@ -41,16 +41,16 @@ module.exports = React.createClass({
   priceNode: function() {
     if (this.props.price === null) return null;
     if (this.props.price === 0) {
-      return this.props.freeText;
+      return <ReactIntl.FormattedMessage id="free" description="The adjective free" defaultMessage="free" />;
     }
-    return (
-      <span><span className="component-basket-item-currency">{this.props.currencySymbol}</span><span className="component-basket-item-price">{this.props.price}</span></span>
-    );
+    // TODO: Revisit this when this issue has been completed: https://github.com/yahoo/react-intl/issues/215
+    // need to pass in `format="price"` and remove `style` and `currency` props.
```

Comment:
yes, just so that comment doesn't get ignored, that we need to come back here and make a change when react-intl is updated